components/com_kunena/site/template/crypsis/layouts/message/item/top/default.php

```diff
@@ -17,21 +17,25 @@
 $attachments = $message->getAttachments();
 $avatarname = $this->profile->getname();
 $topicStarter = $this->topic->first_post_userid == $this->message->userid;
+if (KunenaConfig::getInstance()->ordering_system == 'mesid') {
```

Comment:
good practice to leave long variable usage like this in a variable at the top.

in other pages in practice we are using a variable like $config = KunenaConfig::getInstance()

Then in the actual line we are using $config->orderingsystem